### PR TITLE
Add drone cap parameter

### PR DESCRIPTION
**Pull request by jackcmay**
#### Problem

No way to specify lamport cap when starting the drone

#### Summary of Changes

Fixes #

### CODE REVIEW

File: multinode-demo/drone.sh [hunk 1]
```diff
@@ -7,18 +7,40 @@ here=$(dirname "$0")
...
+    exitcode=1
+    echo "Error: $*"
   fi
-  echo "usage: $0]"
-  echo
-  echo " Run an airdrop drone"
-  echo
-  exit 1
+  cat <<EOF
+usage: $0 [-c lamport_cap]
```
Comment (mvines) [inline multinode-demo/drone.sh]: -n or -c?

File: multinode-demo/drone.sh [hunk 2]
```diff
@@ -7,18 +7,40 @@ here=$(dirname "$0")
...
+
+ -n lamport_cap    - Airdrop cap in lamports
+
+EOF
+  exit $exitcode
 }
 
+lamport_cap=1000000000
+
+while getopts "h?c:" opt; do
```
Comment (mvines) [inline multinode-demo/drone.sh]: How about we just pass through all additional arguments directly to the real drone and nuke all this getopt wrapper stuff

Comment (jackcmay) [reply multinode-demo/drone.sh]: @mvines How's this?

Comment (mvines) [reply multinode-demo/drone.sh]: It looks like we can remove that `usage` function entirely

**Comment by CriesofCarrots** [comment]
@jackcmay , if you rebase on master, it should resolve the `test_replay` panic in CI.

**Pull request merged**